include/zim/zim.h

```diff
@@ -75,6 +75,8 @@ namespace zim
     // This must be the last one and denotes the count of all checks
     COUNT
   };
+
+  void printVersions();
```

Comment:
The name of the function contradicts the commit message where it was introduced

Reply:
OK, see next fixup commit.

Reply:
I have renamed the commit